openedx/openedx-platform
# SE-3247 Allow custom node env variables for plugins

One of our clients require to be able to send customizable values to their custom tinymce plugin that they want to use in studio. This PR adds the ability to share additional node environment variables that are set in django settings file with any js plugins. They can then be accessed via `process.env.<variable-name>`.

**JIRA tickets:** [OSPR-4988](https://openedx.atlassian.net/browse/OSPR-4988)

**Sandbox URL:** TBD - sandbox is being provisioned

**Testing Instructions:**
1. Set any required environment variables inside the `ADDITIONAL_NODE_ENV_VARS` variable in cms/envs/common.py file.
2. Verify that you can access those environment variables from within js code (can try within any of the active tinymce plugins). You can get the value of the environment variables by parsing `process.env.ADDITIONAL_NODE_ENV_VARS`.

**Reviewers:**
